backend/app/services/implementations/log_records_service.py

```diff
@@ -25,13 +25,26 @@ def add_record(self, log_record):
         new_log_record = log_record
         new_log_record["datetime"] = datetime.now()
 
+        tag_names = new_log_record["tags"]
```

Comment:
not sure what specifically needs to change, but the response after creating a log record doesn't have the tags:
```
{
    "attn_to": 1,
    "building": "Building D",
    "datetime": "Tue, 13 Jun 2023 19:15:40 GMT",
    "employee_id": 1,
    "flagged": false,
    "note": "This is a note",
    "resident_first_name": "Saf",
    "resident_last_name": "Waan"
}
```